packages/backend/src/payment-method/handler/service.ts

```diff
@@ -4,8 +4,11 @@ import { WalletAddress } from '../../open_payments/wallet_address/model'
 import { Receiver } from '../../open_payments/receiver/model'
 import { BaseService } from '../../shared/baseService'
 import { IlpPaymentService } from '../ilp/service'
+import { LocalPaymentService } from '../local/service'
+import { Transaction } from 'objection'
 
 export interface StartQuoteOptions {
+  quoteId?: string
```

Comment:
its only used by the ilp implementation (for the `ilpQuoteDetails`). the ilp `getQuote` throws an error if not provided. I figure we dont want to require it for the local `getQuote` since its not used.

It's a little loose type-wise but I don't think there is a better way unless we make more fundamental changes to the payment handler or something.

Reply:
Makes sense, we can keep it as optional in that case